akita/src/transaction.ts

```diff
@@ -114,7 +114,7 @@ export function transaction() {
  * )
  *
  */
-export function withTransaction<T>(transactionFn: Function) {
+export function withTransaction<T>(next: (value: T) => void) {
```

Comment:
You clearly didn't test that it worked. I've reverted it.

Reply:
As stated I only changed it in the browser. I have been using this typing inside my node_modules folder. Its the last time I'll try todo a quick change directly in the web UI and will move to properly clone moving forward. Apologize for the snafu.